server/database/models/orderItem.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../connection.js');
+const { INTEGER } = require('sequelize');
+
+const OrderItem = db.define('orderItem', {
+  id: {
+    type: INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
```

Comment:
What is this meant to do? Sequelize will automatically give an incremented primary key.